tekton/resources/nightly-release/overlays/workflows/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+namePrefix: workflows-
+bases:
+  - ../../base
+patchesJson6902:
+  - target:
+      group: triggers.tekton.dev
+      version: v1alpha1
+      kind: TriggerTemplate
+      name: template
+    path: template.yaml
+  - target:
+      group: triggers.tekton.dev
+      version: v1alpha1
+      kind: Trigger
+      name: nightly
+    path: trigger.yaml
+resources:
+  - github.com/tektoncd/experimental/tekton/?ref=main
```

Comment:
This is specific to custom tasks, right? 
With remote resolution, we may be able to build a catalogue of reusable tasks for release purposes, but we'll have to see if they can be parametrised enough to work for many different projects.

Reply:
I think it's called "publish-customtask" but it should work on anything that has the generated reconciler and webhook set up (doesn't necessarily need to reconcile CustomRuns). I just added a reconciler and webhook for the workflows project so it seems like a natural choice to use this release task. There is one issue though which is being discussed in https://github.com/tektoncd/experimental/pull/902